src/github.com/getlantern/appdir/appdir.go

```diff
@@ -0,0 +1,28 @@
+// package appdir provides a facility for determining the system-dependent
+// paths for application resources.
+package appdir
```

Comment:
appdir is a new library for determining system-dependent paths for app resources.